installers/linux/shared/partials/_go-agent_post_install_message.sh.erb

```diff
@@ -20,4 +20,10 @@
     echo "Now please edit /usr/share/go-agent/wrapper-config/wrapper-properties.conf and set the server to the url address of your GoCD Server (https://example.com:8154/go)."
     echo "To ensure full end-to-end transport security between the agent and server"
     echo "see https://docs.gocd.org/current/installation/ssl_tls/end_to_end_transport_security.html"
-    echo "Once that is done start the GoCD Agent with '/etc/init.d/go-agent start'"
+    if [ -f "/etc/init.d/go-agent" ]; then
+      echo "Once that is done start the GoCD Agent with '/etc/init.d/go-agent start'"
+    elif [ -f "/etc/systemd/system/go-agent.service" ]; then
+      echo "Once that is done start the GoCD Agent with 'systemctl start go-agent.service'"
```

Comment:
I think it's just supposed to be `systemctl start go-{agent,server}`.

Reply:
@ketan -- yes will change!

Reply:
Just a question, as this message only shows up on the agent installation (similar one on server installation). Is there a need to specify both `{agent,server}` ?

Reply:
I meant to say `systemctl start go-agent` in the agent post install, and `systemctl start go-server` in the server post install. Sorry, should have been clearer :)